content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -37,6 +37,12 @@ The upgrade workflow at high level is the following:
 - Make sure you read the [release notes]({{< latest-release-notes >}}) carefully.
 - Make sure to back up any important components, such as app-level state stored in a database.
   `kubeadm upgrade` does not touch your workloads, only components internal to Kubernetes, but backups are always a best practice.
+- If you're running CentOS, RHEL, Fedora or directly use the
+  [yum.k8s.io](https://yum.k8s.io) repositories: The `kubernetes-cni` package
+  was obsoleted with the release of Kubernetes v1.18.4, v1.17.7 and v1.16.11 on
+  June 17th, 2020. This means in case of doing an upgrade from prior versions,
+  please ensure that the package gets **manually** removed before doing the
+  upgrade by executing `yum remove kubernetes-cni-0.7.5-0`.
```

Comment:
@saschagrunert I've thought about this a bit more. I can't test this because I don't have any custom RPM repo, so this is based on the assumption and has to be verified.

Looking at the output from your PR to fix packages (https://github.com/kubernetes/release/pull/1367#issuecomment-646107133), once the `kubernetes-cni` package is gone, `kubelet` will be gone too, and mostly like `kubeadm` as well (because `kubeadm` has a dependency on `kubernetes-cni`).

`kubeadm` is not a big deal because the first step is to install the latest one, but `kubelet` is supposed to be installed after `kubeadm upgrade` (as per docs). If there's no kubelet on the instance, can `kubeadm upgrade` work or is it going to fail? I'd say it might fail because there's no kubelet to schedule all the new static pods, but I'm not sure about that.

If that's true, we might have to update docs to mention this as well.